src/app.ts

```diff
@@ -46,6 +60,27 @@ app.use(validator({
   schemaPath: path.join(__dirname, '../node_modules/@badgateway/tt-types/schema')
 }));
 
+// a12n setup
+const client = new OAuth2Client({
+  server: 'http://localhost:8531',
+  clientId: 'tt-api',
+  clientSecret: process.env.OAUTH2_CLIENT_SECRET,
+  tokenEndpoint: '/token',
+  introspectionEndpoint: '/introspect',
```

Comment:
These 2 don't need to be specified, they should get auto-detected if the `server` is specified correctly.